common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -148,6 +148,10 @@ div.problem {
       margin-top: $baseline;
     }
   }
+
+  .MathJax_Preview, .MathJax_CHTML {
```

Comment:
Considering that MathJax allows the user to choose their own renderer, and upon doing so, MathJax changes the class name, we should probably add the following classes to this definition:
```.MathJax, .MathJax_CHTML, .MathJax_PHTML, .MathJax_MathML, MathJax_SVG, .MathJax_PlainSource```